### ISSUE TITLE
posts

### ISSUE THREAD

**Issue by nnop** [nnop/notes#18]
rng

- http://blogs.unity3d.com/2015/01/07/a-primer-on-repeatable-random-numbers/
- http://stackoverflow.com/questions/322938/recommended-way-to-initialize-srand
- https://www.guyrutenberg.com/2007/09/03/seeding-srand/
- http://stackoverflow.com/questions/20636859/random-number-generator-why-seed-every-time
- http://stackoverflow.com/questions/7343833/srand-why-call-it-only-once

**Comment by nnop** [comment]
# RNN
- http://r2rt.com/
- http://www.wildml.com/2015/10/recurrent-neural-networks-tutorial-part-3-backpropagation-through-time-and-vanishing-gradients/
- http://jefkine.com/general/2016/09/05/backpropagation-in-convolutional-neural-networks/

# Lua
- http://tylerneylon.com/a/learn-lua/
- http://www.phailed.me/2011/02/learn-lua-the-hard-way-1/
- http://luatut.com/introduction.html
- http://www.tutorialspoint.com/lua/

**Comment by nnop** [comment]
STN 

- Supervised Transformer Network for Efficient Face Detection

**Comment by nnop** [comment]
# Detection

- Inside-Outside Net: Detecting Objects in Context with Skip Pooling and Recurrent Neural Networks

**Comment by nnop** [comment]
CTC implementation
- http://www.jianshu.com/p/4fadf629895b
- https://zhuanlan.zhihu.com/p/21344595
- https://zhuanlan.zhihu.com/p/23664023
